src/clndr.js

```diff
@@ -612,7 +618,7 @@
         } else {
           target.events = $.makeArray( $(this.options.events).filter( function() {
             // filter the dates down to the ones that match.
-            return this._clndrDateObject.format('YYYY-MM-DD') == dateString;
+            return this._clndrDateObject.format('YYYY-MM-DD') == target.date;
```

Comment:
I think the target.date is a moment object and would need a similar .format() call?